### PR TITLE
Fix Makefile and error in Stacey GPU implementation

### PR DESCRIPTION
**Pull request by EtienneBachmann**

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
The files changed require a buildbot test. Buildbot has started.

**Comment by buildbot-princeton** [comment]
Your changes passed the buildbot test. See [build 345](http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/345).

**Comment by komatits** [comment]
accept

On 11/30/2015 02:44 AM, buildbot-princeton wrote:
> Your changes passed the buildbot test. See build 345
> <http://specfem2d-buildbot.princeton.edu/builders/specfem2d_builder/builds/345>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geodynamics/specfem2d/pull/493#issuecomment-160495990>.
>

-- 
Dimitri Komatitsch
CNRS Research Director (DR CNRS), Laboratory of Mechanics and Acoustics,
UPR 7051, Marseille, France    http://komatitsch.free.fr

File: src/specfem2D/prepare_timerun_gpu.f90 [hunk 1]
```diff
@@ -384,88 +384,105 @@ subroutine init_host_to_dev_variable()
            cote_abs(nelemabs),stat=ier)
   if (ier /= 0 ) stop 'error allocating array abs_boundary_ispec etc.'
 
-  ! stacey boundaries
-  if (STACEY_BOUNDARY_CONDITIONS) then
-    ! user output
-    if (myrank == 0) write(IMAIN,*) '    Stacey boundary elements = ',nelemabs
+   if(STACEY_BOUNDARY_CONDITIONS) then
```
Comment (danielpeter) [inline src/specfem2D/prepare_timerun_gpu.f90]: hi etienne,

please use a consistent indenting with 2-spaces for the next indent level. your indenting here is all messed up and makes it difficult to read the code and to see in which if/do-statement things are done. please make sure your editor uses spaces and not tabs for indenting.

many thanks,
daniel

**Pull request merged**